cmd/flow-dps-indexer/main.go

```diff
@@ -73,13 +78,15 @@ func run() int {
 	pflag.BoolVarP(&flagForce, "force", "f", false, "overwrite existing index database")
 	pflag.StringVarP(&flagIndex, "index", "i", "index", "database directory for state index")
 	pflag.BoolVarP(&flagIndexAll, "index-all", "a", false, "index everything")
-	pflag.BoolVarP(&flagIndexCollections, "index-collections", "o", false, "index collections")
-	pflag.BoolVarP(&flagIndexCommit, "index-commits", "m", false, "index commits")
-	pflag.BoolVarP(&flagIndexEvents, "index-events", "e", false, "index events")
-	pflag.BoolVarP(&flagIndexHeader, "index-headers", "h", false, "index headers")
-	pflag.BoolVarP(&flagIndexPayloads, "index-payloads", "p", false, "index payloads")
-	pflag.BoolVarP(&flagIndexTransactions, "index-transactions", "x", false, "index transactions")
+	pflag.BoolVar(&flagIndexCollections, "index-collections", false, "index collections")
+	pflag.BoolVar(&flagIndexCommit, "index-commits", false, "index commits")
+	pflag.BoolVar(&flagIndexEvents, "index-events", false, "index events")
+	pflag.BoolVar(&flagIndexHeader, "index-headers", false, "index headers")
+	pflag.BoolVar(&flagIndexPayloads, "index-payloads", false, "index payloads")
+	pflag.BoolVar(&flagIndexTransactions, "index-transactions", false, "index transactions")
```

Comment:
Remind me to make the same change to the added indexes when we merge the access API feature branch :p